test/lsptoolshost/integrationTests/sourceGenerator.integration.test.ts

```diff
@@ -50,11 +50,11 @@ describe(`Source Generator Tests`, () => {
                 requestPosition
             )
         );
-        expect(referencesList.length).toEqual(8);
+        expect(referencesList.length).toEqual(5);
```

Comment:
Building against the net8.0 tfm causes the JsonSourceGenerator to generate fewer references.